core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -6274,6 +6274,65 @@ private void checkNullOperand(SqlTester tester, String op) {
     tester.checkNull("substring(cast(null as varchar(1)),1,2)");
   }
 
+  @Test public void testRegexpSplitToArrayFunction() {
+    final SqlTester t = tester(SqlLibrary.POSTGRESQL);
+    t.setFor(SqlLibraryOperators.REGEXP_SPLIT_TO_ARRAY);
+    t.checkString(
+        "regexp_split_to_array('','a')",
+        "[]",
```

Comment:
Can we also add some tests for implicit type coercion, for example `regexp_split_to_array(1245, '2')`.

Reply:
OK.